githubteacher-tampa.md

```diff
@@ -0,0 +1,5 @@
+## Places to Eat
+ - Dunedin Brewery :beer:
+ - Demitri's on the Water
+   - Eat outside, feed the fish / :duck:
+ - Thai Restaurant on Sunset Blvd (Clearwater)
```

Comment:
This is not helpful. I hope in your next iteration you provide the actual restaurant name.